lib/api/controllers/securityController.js

```diff
@@ -0,0 +1,79 @@
+var
+  ResponseObject = require('../core/models/responseObject');
+
+module.exports = function SecurityController (kuzzle) {
+
+  /**
+   * Get a specific role according to the given id
+   * @param {RequestObject} requestObject
+   * @returns {Promise}
+   */
+  this.getRole = function (requestObject) {
+    kuzzle.pluginsManager.trigger('data:getRole', requestObject);
+
+    return kuzzle.repositories.role.loadRole(
+      kuzzle.repositories.role.getRoleFromRequestObject(requestObject)
+      )
+      .then(role => {
+        if (role && role.closures) {
+          delete role.closures;
+        }
+
+        return Promise.resolve(new ResponseObject(requestObject, role || {}));
```

Comment:
should return `q(new ResponseObject...` to harmonize the promise objects we use throughout the entire project